README.md

```diff
@@ -6,25 +6,3 @@
 0. [install test dependencies](https://github.com/rackerlabs/helix-ui/wiki/Testing#system-requirements)
 0. `yarn install` to install project dependencies
 0. `yarn start` to begin working
-
-
-### Key Files/Directories
```

Comment:
Removed. This section hasn't been correct for a while. If we need to document the yarn tasks, we'll do so in the project wiki.